dark.css

```diff
@@ -1,5 +1,33 @@
+/* 深色模式下的样式 */
 @media (prefers-color-scheme: dark) {
   img.invert-on-dark-mode {
     filter: invert(1);
   }
+
+  h4 {
+    color: #BBEFFC;
+  }
+
+  #langSwitch button {
+    background: #627262 none repeat scroll 0% 0%;
+    border: 1px solid #CCC;
+    border-radius: 5px;
+  }
+
+  #langSwitch button:lang(zh-hans) {
+    background-color: #005a9c;
+  }
+
+  #langSwitch button:lang(zh-hant) {
+    background-color: #0171c4;
+  }
```

Comment:
These two colours are too close to each other. The different colours here are to allow people who do not understand Chinese to better distinguish Traditional and Simplified Chinese.

In addition, we need to update https://github.com/w3c/clreq/blob/7d515699e5d805ddf7183173ab910a1d294ddf65/local.css#L183-L188 to make them consistent.